### ISSUE TITLE
`scw cp` error

### ISSUE THREAD

**Issue by nicolas-brousse** [scaleway/scaleway-cli#395]
I try to use `scw cp` but I have an error. I try examples provided from `scw htop cp` and no one works.

**what I do**
```sh
$ scw cp ./ my-server:/opt/cookbooks
FATA[0001] cannot execute 'cp': cannot untar to destination 'my-server:/opt/cookbooks': write |1: broken pipe 
```

**environment**
* local OS: OSX
* server image: Debian_Jessie
* scw version: `scw version v1.9.0, build homebrew`

**I also tried**
```sh
$ tar cj . | scw exec my-server "cd /opt/cookbooks && tar zj"
FATA[0001] cannot execute 'exec': Failed to run the command: exit status 255

$ tar zcf - . | scw exec my-server "cd /opt/cookbooks && tar zxf -"
FATA[0000] cannot execute 'exec': Failed to run the command: exit status 255 

$ tar -cvf - . | scw cp - my-server:/opt/cookbooks # Works but sub-folders are empty and not work for a file
```

**Comment by QuentinPerez** [comment]
Hi @nicolas-brousse,

I am working on your issue and for your first case, you can't use `-j` with `-z`

```console
$ tar zj
tar: Can't specify both -j and -z
$ tar cj . | scw exec vps "cd /tmp/ && tar xj" # use xj instead of zj
bzip2: (stdin): trailing garbage after EOF ignored
$ scw exec vps "ls /tmp"
file1  file2  file3
```

the second and last work fine for me :/ can you give me your version with `scw version` ?

```console
$ scw exec vps "rm -fr /tmp/*"
$ scw exec vps "ls /tmp/"
$ tar zcf - . | scw exec vps "cd /tmp && tar zxf -"
$ scw exec vps "ls /tmp/"
file1  file2  file3
```

```
$ tar -cvf - . | scw cp - vps:/tmp
$ scw exec vps "ls -lR /tmp/"
/tmp/:
total 120
drwxr-xr-x 2 501 root   4096 Sep  2 18:40 dir
-rw-r--r-- 1 501 root   2048 Sep  2 18:21 file1
-rw-r--r-- 1 501 root  10240 Sep  2 18:21 file2
-rw-r--r-- 1 501 root 102400 Sep  2 18:21 file3

/tmp/dir:
total 100
-rw-r--r-- 1 501 root 102400 Sep  2 18:40 file4
```

**Comment by nicolas-brousse** [comment]
Hi,

**scw version**
```sh
➜ ~ % scw version
Client version: v1.9.0
Go version (client): go1.6
Git commit (client): homebrew
OS/Arch (client): darwin/amd64
```

I'm starting a scaleway vps to do some tests

**Comment by QuentinPerez** [comment]
Can you clone this repo and test with the latest version ?

**Comment by nicolas-brousse** [comment]
You right for `tar zj` it's a typo..

**with scw stable from homebrew**
```sh
➜ ~ % scw version
Client version: v1.9.0
Go version (client): go1.6
Git commit (client): homebrew
OS/Arch (client): darwin/amd64

➜ ~ % tar cj . | scw exec vps "cd /tmp/ && tar xj" 
^C"?$?ELJ^EA!^R
"?$?ELJ^EA!2?)B???^LL#KAH?DI$?A^@I2L?J^P2J$R^L??V^K??!/^FATA[0000] cannot execute 'exec': Failed to run the command: exit status 255 

➜ ~ % scw exec vps "ls /tmp/"
file1  file2  file3
```
I still have an error :/

**with scw --HEAD from homebrew (I think it's the same than clone repo)**
```sh
➜ ~ % scw version
Client version: v1.9.0+dev
Go version (client): go1.7
Git commit (client): homebrew
OS/Arch (client): darwin/amd64

➜ ~ % tar cj . | scw exec vps "cd /tmp/ && tar xj" 

bzip2: (stdin): trailing garbage after EOF ignored

➜ ~ % tar zcf - . | scw exec vps "cd /tmp && tar zxf -"
➜ ~ % scw exec vps "ls /tmp/"
file1  file2  file3
```

Well looks like all commands works with dev version :)

**Comment by QuentinPerez** [comment]
Nice 👍

**Comment by nicolas-brousse** [comment]
I guess it will be "fixed" when 1.10.0 will be released?
For now I will use `--HEAD` version :)

**Comment by QuentinPerez** [comment]
Yes, I will try to close some issue this week end to release the 1.10 😊
